grid/chat/send-message/maps/send-message.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable jest/no-export */
+
+import { RecordingProcessOptions, SuperfaceTest } from '@superfaceai/testing';
+
+export const sendMessageTest = (
+  provider: string,
+  destination: string[],
```

Comment:
TBH this API doesn't tell anything about the fact that the second destination is supposed to be non-existent. As a quick workaround, you can define this parameter as a tuple, but consider better function interfaces in the future.